### PR TITLE
fix(links): update line height for the links

### PR DESCRIPTION
**Pull request by elenagarrone**
# Description

Fix an issue where the inline block links had different line height than the inline ones.
Also the inline block links on hover were causing the below content to shift.

Behaviour before:

https://user-images.githubusercontent.com/8397116/124100493-6e72ec00-da56-11eb-9f1a-cbdb6278f703.mov

Behaviour after:

https://user-images.githubusercontent.com/8397116/124100090-0fad7280-da56-11eb-9638-3598df791685.mov

Storybook link: https://deploy-preview-442--legal-and-general-canopy.netlify.app/?path=/story/links--utils

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [x] I have provided a story in storybook to document the changes
- [x] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts

### CODE REVIEW

**Review by paul-request — DISMISSED** [review]

File: projects/canopy/src/styles/mixins.scss [hunk 1]
```diff
@@ -24,10 +24,11 @@
 ) {
   $box-shadow-inset-width: -0.063rem;
 
-  color: $default-color;
-  text-decoration: none;
   border-bottom: 0.125rem solid $default-color;
+  color: $default-color;
+  line-height: var(--link-line-height);
```
Comment (paul-request) [inline projects/canopy/src/styles/mixins.scss:29]: Or should it just inherit from the parent?

**Comment by elenagarrone** [comment]
@andy-polhill, @owensgit and @paul-request this might be a breaking change as some people might have changed their code to fix this? Do you know if this was the case?

File: projects/canopy/src/styles/mixins.scss (hunk 1 above)
Comment (elenagarrone) [reply projects/canopy/src/styles/mixins.scss:29]: I tried that but it doesn't seem to work 🤔

Comment (elenagarrone) [reply projects/canopy/src/styles/mixins.scss:29]: Here is using `inherit`: 
Screenshot 2021-07-01 at 10 27 50

Comment (paul-request) [reply projects/canopy/src/styles/mixins.scss:29]: I was just wondering about a case where someone modifies the font-size of the parent with say, `lg-font-size-2` which contains a link, would that break the layout?

Comment (elenagarrone) [reply projects/canopy/src/styles/mixins.scss:29]: BUT `initial` works so I'll change it to that 🤔

Comment (paul-request) [reply projects/canopy/src/styles/mixins.scss]: Oh, I see. It is about where the "underline" sits... not sure initial is what we actually want. maybe the actual variable is the best solution

Comment (elenagarrone) [reply projects/canopy/src/styles/mixins.scss]: Updated to `initial` now :)

**Review by paul-request — APPROVED** [review]

**Comment by aaronBery** [comment]
Screenshot 2021-07-01 at 16 29 34

@elenagarrone  just to check...is link meant to sit under utils as the top level navigation item or was it intended this way?

**Comment by elenagarrone** [comment]
> Screenshot 2021-07-01 at 16 29 34
> 
> @elenagarrone just to check...is link meant to sit under utils as the top level navigation item or was it intended this way?

Definitely not, how weird - I'll check what's going on 👍

**Comment by elenagarrone** [comment]
@aaronBery thanks I have now fixed that 👍

**Review by aaronBery — APPROVED** [review]

**Review by andy-polhill — APPROVED** [review]

**Comment by owensgit** [comment]
Just saw this, love the video clips in the description 🙂

**Pull request merged**